core/shared/src/main/scala/org/http4s/internal/package.scala

```diff
@@ -36,92 +36,9 @@ import java.util.concurrent.CancellationException
 import java.util.concurrent.CompletableFuture
 import java.util.concurrent.CompletionException
 import java.util.concurrent.CompletionStage
-import scala.util.control.NoStackTrace
 
 package object internal extends InternalPlatform {
 
-  /** Hex encoding digits. Adapted from apache commons Hex.encodeHex */
-  @deprecated("Will be removed in 1.0.", "0.23.19")
-  private val Digits: Array[Char] =
-    Array('0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'A', 'B', 'C', 'D', 'E', 'F')
-
-  /** Encode a byte Array into a hexadecimal string
-    *
-    * @param data the array
-    * @return a hexadecimal encoded string
-    */
-  @deprecated("Will be removed in 1.0.", "0.23.19")
-  private[http4s] final def encodeHexString(data: Array[Byte]): String =
-    new String(encodeHex(data))
-
-  /** Encode a string to a Hexadecimal string representation
-    * Adapted from apache commons Hex.encodeHex
-    */
-  @deprecated("Will be removed in 1.0.", "0.23.19")
-  private[http4s] final def encodeHex(data: Array[Byte]): Array[Char] = {
-    val l = data.length
-    val out = new Array[Char](l << 1)
-    // two characters form the hex value.
-    def iterateData(out: Array[Char], l: Int): Array[Char] = {
-      def innerEncode(l: Int, i: Int, j: Int): Array[Char] =
-        i match {
-          case k if k < l =>
-            out(j) = Digits((0xf0 & data(k)) >>> 4)
-            out(j + 1) = Digits(0x0f & data(k))
-            innerEncode(l, k + 1, j + 2)
-          case _ => out
-        }
-      innerEncode(l, 0, 0)
-    }
-    iterateData(out, l)
-  }
-
-  @deprecated("Will be removed in 1.0.", "0.23.19")
-  private[http4s] final def decodeHexString(data: String): Option[Array[Byte]] =
-    decodeHex(data.toCharArray)
-
-  @deprecated("Will be removed in 1.0.", "0.23.19")
-  private object HexDecodeException extends Exception with NoStackTrace
-
-  /** Dirty, optimized hex decoding based off of apache
-    * common hex decoding, ported over to scala
-    *
-    * @param data
-    * @return
-    */
-  @deprecated("Will be removed in 1.0.", "0.23.19")
-  private[http4s] final def decodeHex(data: Array[Char]): Option[Array[Byte]] = {
-    def toDigit(ch: Char): Int = {
-      val digit = Character.digit(ch, 16)
-      if (digit == -1)
-        throw HexDecodeException
-      else
-        digit
-    }
-
-    val len = data.length
-    if ((len & 0x01) != 0) None
-    val out = new Array[Byte](len >> 1)
-    var f: Int = -1
-    // two characters form the hex value.
-    try {
-      var i = 0
-      var j = 0
-      while (j < len) {
-        f = toDigit(data(j)) << 4
-        j += 1
-        f = f | toDigit(data(j))
-        j += 1
-        out(i) = (f & 0xff).toByte
-
-        i += 1
-      }
-      Some(out)
-    } catch {
-      case HexDecodeException => None
-    }
-  }
-
   private[http4s] def fromCompletionStage[F[_], CF[x] <: CompletionStage[x], A](fcs: F[CF[A]])(
```

Comment:
I'd love to remove this too. Can't find if anyone is using it, and now we have `fromCompletableFuture` in Cats Effect.

Reply:
I neither succeeded in finding usages of these, so let's do that!